### PR TITLE
Resource section

### PR DESCRIPTION
**Pull request by shans**

### CODE REVIEW

File: runtime/manifest-parser.peg [hunk 1]
```diff
@@ -72,8 +72,23 @@ ManifestItem
...
+  / Resource
 
-Annotation = '@' annotation: lowerIdent { return annotation; }
+Annotation = '@' annotation:lowerIdent { return annotation; }
+
+Resource = 'resource' whiteSpace name:TopLevelIdent eolWhiteSpace Indent SameIndent 'start' eolWhiteSpace body:ResourceBody {
+  return {
+    kind: 'resource',
+    name,
+    data: body
```
Comment (dstockwell) [inline runtime/manifest-parser.peg]: hmm, this should include `location`, `checkNormal` is supposed to error on this being absent. Looks like this change accidentally added 'object': https://github.com/PolymerLabs/arcs/pull/575/files#diff-b8b73ba247e6fb5f18a68ca683419175R27

File: runtime/manifest-parser.peg [hunk 2]
```diff
@@ -827,6 +842,8 @@ eolWhiteSpace
   = [ ]* !.
   / [ ]* "\r"? "\n" "\r"? eolWhiteSpace?
```
Comment (dstockwell) [inline runtime/manifest-parser.peg]: use eol here?

File: runtime/manifest-parser.peg (hunk 1 above)
Comment (shans) [reply runtime/manifest-parser.peg]: Reverted that and fixed the failing instances.

File: runtime/manifest-parser.peg (hunk 2 above)
Comment (shans) [reply runtime/manifest-parser.peg]: Done.

**Pull request merged**